src/rdbms-connect/azext_rdbms_connect/custom.py

```diff
@@ -9,32 +9,74 @@
 from knack.log import get_logger
 import psycopg2
 import pymysql
+from pymysql.constants import CLIENT
+from azure.cli.core.azclierror import RequiredArgumentMissingError, AzureConnectionError
+from azure.cli.core.extension import EXTENSIONS_DIR
+
 
 logger = get_logger(__name__)
 DEFAULT_MYSQL_DB_NAME = ''  # no default database required
 DEFAULT_PG_DB_NAME = 'postgres'
 
 
 def connect_to_flexible_server_mysql(cmd, server_name, administrator_login, administrator_login_password=None,
-                                     database_name=None, querytext=None, interactive_mode=None):
```

Comment:
`querytest` is not supported in your command?

Reply:
Same as above. query test is moved to execute command group. The execute command group will take care of file execution as well as query execution